src/marketing/customer-segment-dynamic-block.md

```diff
@@ -0,0 +1,36 @@
+---
+ee_only: true
+title: Customer Segments for Dynamic Blocks
+group: customers
+redirect_from:
+  - /marketing/customer-segment-banner.html
+---
+
+A [dynamic block]({% link cms/dynamic-blocks.md %}) can target a specific [customer segment]({% link marketing/customer-segments.md %}), and be incorporated into a [cart price rule]({% link marketing/price-rules-cart.md %}). Because customer segments are dynamic, the price rule can adjust to changes in customer activity.
+
+![]({% link images/images-ee/page-builder-tutorial2-dynamic-block-storefront.png %}){: .zoom}
+_Dynamic Block with price rule and customer segment_
+
+## Associate a customer segment with a dynamic block
+
+1. On the _Admin_ sidebar, go to **Content** > _Elements_ > **Dynamic Blocks**.
+
+1. Open a new or existing dynamic block:
+
+   - To use a new banner, click <span class="btn">Add Dynamic Block</span> in the upper-right corner.
```

Comment:
should this say banner or dynamic block?

Reply:
Good catch!  Should be dynamic block. Will fix :)